### ISSUE TITLE
Getting ES Stats queries all indexes, throws error for limited ES user.

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#20596]
When running ES in a multi-tenant environment, we want to isolate and permission each customer to only be able to see and stat the indexes that belong to them and their cluster.  In our method `ESIndexAPI.getIndicesStats()`, we call for the stats of all indexes, which a limited ES user will not have permissions to do.  We do:

```
final Request request = new Request("GET", "/_stats");
```
and what we should do is:
```
final Request request = new Request("GET", "/" + clusterPrefix.get() + "*/_stats");
```

**Comment by nollymar** [comment]
**Internal QA:** Passed

**Comment by bryanboza** [comment]
Fixed, tested on release 21.10 // Docker // FF 

Tested with the provided example, replacing the clusterPrefix.get() by the cluster id.
